### PR TITLE
Admin stats page

### PR DESCRIPTION
**Pull request by nakedmcse**
Add Stats page with graphs of useful overall information about the site.

### CODE REVIEW

File: src/controllers/secure/AdminView.ts [hunk 1]
```diff
@@ -2,15 +2,33 @@ import { Get, Hidden, View } from "@tsed/schema";
...
+            user: req.user as CustomUserInfoModel,
+        };
+    }
+
+    @Get("/stats")
+    @Authorize("loginAuthProvider")
+    @View("/secure/stats.ejs")
+    public showStatistics(@Req() req: Req): unknown {
         return {
             user: req.user as CustomUserInfoModel,
```
Comment (VictoriqueMoe) [inline src/controllers/secure/AdminView.ts:33]: does stats need the user? don't think it does, although can correct me if i am wrong

File: src/controllers/views/HomeView.ts [hunk 2]
```diff
@@ -15,7 +15,7 @@ export class HomeView {
     @View("login.ejs")
     public showLogin(@Req() req: Req, @Res() res: Res): unknown {
         if (req.user) {
-            res.redirect("/admin");
+            res.redirect("/admin/files");
```
Comment (VictoriqueMoe) [inline src/controllers/views/HomeView.ts:18]: login should re-direct to the dashboard you made, not files

**Review by VictoriqueMoe — CHANGES_REQUESTED** [review]

File: src/controllers/secure/AdminView.ts (hunk 1 above)
Comment (nakedmcse) [reply src/controllers/secure/AdminView.ts:33]: Removed

File: src/controllers/views/HomeView.ts (hunk 2 above)
Comment (nakedmcse) [reply src/controllers/views/HomeView.ts:18]: Adjusted

**Review by VictoriqueMoe — APPROVED** [review]

**Pull request merged**